src/libraries/SqrtPriceMath.sol

```diff
@@ -200,8 +200,8 @@ library SqrtPriceMath {
     {
         unchecked {
             return liquidity < 0
```

Comment:
Right the argument is that `liquidityDelta` is the user's delta of _liquidity units_ not token units. When they pass in `liquidityDelta = 100` its because they want their liquidity balance to increase by 100 (which means sending tokens into the contract). So `liquidityDelta` is in line with `amountSpecified` now... for both of them:
- positive: the balance of the thing in question goes up
- negative: the balance of the thing in question goes down